api/EventTarget.json

```diff
@@ -527,6 +527,57 @@
                 "deprecated": false
               }
             }
+          },
+          "signal": {
+            "__compat": {
+              "description": "<code>signal</code> option",
+              "support": {
+                "chrome": {
+                  "version_added": "89"
```

Comment:
@ddbeck FYI you indicated Chrome 90, my testing shows 89 works, but the fact Opera works on Opera 74 indicates maybe it went in on Chrome 88?

Reply:
Nice catch. You must have experimental web features turned on. It was [added behind a preference in 88](https://storage.googleapis.com/chromium-find-releases-static/f0b.html#f0bbc3f3ab2bf91bea606fbc5fda751f5e60a0d3) and [enabled by default in 90](https://storage.googleapis.com/chromium-find-releases-static/a64.html#a641ee4fd048768763e23a6b4126c1990b64f86b).

You can enter them as two support statements (one with a flag added in 88 and removed in 90, then one without a flag added in 90), or you can ignore the flag and only report 90 (since, by the time this lands in BCD, it will be interesting to web developers for a couple of weeks at best).